### DISCUSSION TITLE
Is there a file that defines keybindings?

### DISCUSSION THREAD

**Discussion by jirafey** [mbnuqw/sidebery#1729] [General]
I use NixOS and would like to configure them locally instead of via extension GUI.
 I saw that I can change the CSS in a file, but no information about keybindings was found during my research.

**Comment by cynicaljoy** [comment]
would also love to be able to define tabs panels and rules this way

**Comment by ArtemChandragupta** [comment]
I also handling my firefox config with nix and would be happy to have an option to declare keybindings, style and tabs in a file

**Comment by jirafey** [comment]
@mbnuqw can we get a response from you?

**Reply by cynicaljoy** [reply]
not trying to start an internet fight, just think it's poor form to expect a response. if you _really_ want the issue to get attention from the maintainer they've already said they'd consider it with a [donation](https://github.com/mbnuqw/sidebery?tab=readme-ov-file#donate). If you go that route, I'd be willing to throw in with you. But this is a nice to have for me, not a must have.

**Reply by silverwings15** [reply]
is mbnuqw around these days? i'm under the impression he has been afk for the past few months. hoping to see his return soon!

**Reply by cynicaljoy** [reply]
merged a few PRs in the last 24 hours, but beyond that :man_shrugging: -- hopefully he stepped way for fun reasons and not OSS maintainer fatigue.
